toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss

```diff
@@ -105,6 +105,64 @@
 		font-weight: bold;
 	}
 
+	&__author-modal {
+		background-color: #fff;
+		height: auto;
+		max-height: 95%;
+		margin: 0 20px 0 0;
+		padding: 0;
+		width: auto;
+		max-width: 380px;
+		position: relative;
+		overflow: visible;
+
+		.js & {
+			position: absolute;
+			width: 90vw;
+			border-top: 3px solid #007e99;
+			border: 2px solid #00b8b0;
+			box-shadow: 0 4px 6px 2px rgba(0, 0, 0, 0.2);
+			border-radius: 3px;
+			padding: 30px;
+			margin: -15px 30px 30px 0;
```

Comment:
should these be using the spacing function? e.g. `spacing(30)`

apologies if i've asked this before on another PR and forgotten

Reply:
No idea! Never heard of it until now 😆

Reply:
haha, it's a function in default brand context the purpose of which is to constrain developers to only use spacing values from a defined map per brand - for sake of consistency